crates/neon-build/src/lib.rs

```diff
@@ -22,6 +22,11 @@ cfg_if! {
             println!("cargo:rustc-link-search=native={}", &node_lib_path.display());
             println!("cargo:rustc-link-lib={}", &node_lib_file_path.file_stem().unwrap().to_str().unwrap());
 
+            if debug
+            {
+                println!("cargo:rustc-link-lib=msvcrtd");
```

Comment:
This doesn't appear to be impacting other compilers (tests are passing), but, just to be safe, can you add a `cfg` to target just `msvc`?

Reply:
👍 Yes I agree to your suggestion. I'll try to fixup. Thank you.

Reply:
What is msvcrtd? Is that the debug mode of Microsoft's C runtime library?